lib/onebox/engine/amazon_onebox.rb

```diff
@@ -46,6 +46,37 @@ def http_params
         end
       end
 
+      def to_html(ignore_errors = false)
+        unless ignore_errors
+          verified_data # forces a check for missing fields
+          return '' unless errors.empty?
+        end
+
+        super()
+      end
+
+      def placeholder_html
+        to_html(true)
+      end
+
+      def verified_data
+        @verified_data ||= begin
+          result = data
+
+          required_tags = [:title, :description]
+          required_tags.each do |tag|
+            if result[tag] == nil || result[tag] == ''
```

Comment:
Why not use `blank?` here?

```suggestion
            if result[tag].blank?
```